services/app-api/src/domain-models/user.ts

```diff
@@ -64,13 +66,35 @@ function toDomainUser(user: PrismaUser): UserType {
                 divisionAssignment: user.divisionAssignment,
                 stateAssignments: [],
             } as CMSUserType
+        case 'CMS_APPROVER_USER':
+            return {
+                id: user.id,
+                role: 'CMS_APPROVER_USER',
+                email: user.email,
+                givenName: user.givenName,
+                familyName: user.familyName,
+                divisionAssignment: user.divisionAssignment,
+                stateAssignments: [],
+            } as CMSApproverUserType
         case 'STATE_USER':
             return user as StateUserType
     }
 }
 
-function hasAdminPermissions(user: UserType): boolean {
-    const authorizedAdmins = [
+// Users with CMS reviewer and approver permissions
+function hasCMSPermissions(
+    user: UserType
+): user is CMSUserType | CMSApproverUserType {
+    const authorizedCMSUsers: UserRoles[] = ['CMS_USER', 'CMS_APPROVER_USER']
+
+    return authorizedCMSUsers.includes(user.role)
+}
+
+// Users with admin permissions
+function hasAdminPermissions(
+    user: UserType
+): user is AdminUserType | HelpdeskUserType | BusinessOwnerUserType {
+    const authorizedAdmins: UserRoles[] = [
         'ADMIN_USER',
         'HELPDESK_USER',
         'BUSINESSOWNER_USER',
```

Comment:
Is `BUSINESSOWNER_USER` supposed to have `Admin` persmissions? Looking at this [IDM roles doc](https://docs.google.com/document/d/1XeVbEimaNLwhVAa9rlPCVVnMDer22Blz/edit#heading=h.tyjcwt) seems like it's more fine-grained than just full admin access.

Reply:
I don't *think* so, but I'm not confident. 

 @macrael or @MegMurphy do you know? Admin permissions for the frontend means they can see all the UI that the Admin user can for support or validation reasons.

Reply:
businesowner is Junni, I think giving her admin permissions is right

Reply:
yes, BO should have admin permissions. Only difference btwn BO and Admin is what type of role requests they can approve in IDM. Junni is the BO role, and she can have admin rights.